proc/api.thrift

```diff
@@ -0,0 +1,40 @@
+service Proc {
+  
+  exception InvalidRequest {
+    1: optional string message
+  }
+
+  exception ScootTooBusy {
+    1: optional i64 retryAfterMs
+  }
+  
+  enum JobType {
+    IRON_TESTS=1,
+    DEFAULT=2,
+  }
+
+  struct Command {
+    1: list<string> cmds
+  }
+
+  struct Task {
+    1: optional string id
+    2: required Command command,
+    3: optional string snapshotId,
+    4: optional string contextId
+  }
+
+  struct Job {
+    1: required string id,
+    2: required list<Task> tasks
+  }
+
+  Job RunJob(
+    1: optional string contextId,
```

Comment:
Yeah, that makes sense. I feel like this is a feature we know we'll need, but we're not 100% clear on the best way to do it.

I'd suggest leaving this until we're ready to start passing meaningful values here, and then it should be much clearer what API we want.